### PR TITLE
Last check

### PR DESCRIPTION
**Pull request by SanderHeidenis**

### CODE REVIEW

File: binarybreakersfrontend/components/custom/dropdown.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ const DropDown = ({array, label, className}: DropDownProps) => {
                     <DropdownMenu>
                         <DropdownMenuTrigger className="min-w-full flex justify-between">
                             <span className="flex-grow-1 mr-3">{selection}</span>
-                            <span className="float-right flex-grow-0">V</span>
+                            <span className="float-right flex-grow-0">V</span>//TODO: Add chevron down icon
```
Comment (Quinten36) [inline binarybreakersfrontend/components/custom/dropdown.tsx]: ͡° ͜ʖ ͡°

**Review by Quinten36 — CHANGES_REQUESTED** [review]
(╯°□°）╯︵ ┻━┻

File: binarybreakersfrontend/components/custom/dropdown.tsx (hunk 1 above)
Comment (SanderHeidenis) [reply binarybreakersfrontend/components/custom/dropdown.tsx]: gave up, removed file. Pushed to prod.
Good luck!

**Review by Quinten36 — APPROVED** [review]
┬─┬ノ( º _ ºノ)

**Pull request merged**